### PR TITLE
Updating packer instructions

### PR DESCRIPTION
**Pull request by carlosbravoa**
Adding the current caveats for using packer with newer Ubuntu versions

### CODE REVIEW

File: aws/aws-how-to/instances/build-pro-ami-using-packer.rst [hunk 1]
```diff
@@ -9,7 +9,7 @@ We'll be using Ubuntu Pro 20.04 for this guide, but the method is equally applic
 
    * For **Ubuntu Pro FIPS**, it is better to use a pre-enabled FIPS image from the Marketplace to avoid unnecessary additional steps. 
    * For **Ubuntu LTS**, you can use this method with a small change as explained at the end of the :ref:`define-provisioners` section below. 
-
+   * For **Ubuntu 22.04 LTS and above** you need packer version 1.8.1 or newer.
```
Comment (toabctl) [inline aws/aws-how-to/instances/build-pro-ami-using-packer.rst:12]: later, you write `1.7.1` is required. which version is it?

**Review by toabctl — CHANGES_REQUESTED** [review]

File: aws/aws-how-to/instances/build-pro-ami-using-packer.rst (hunk 1 above)
Comment (carlosbravoa) [reply aws/aws-how-to/instances/build-pro-ami-using-packer.rst:12]: Good catch! it is 1.8.1. Let me update the docs.

Comment (k-dimple) [inline aws/aws-how-to/instances/build-pro-ami-using-packer.rst:12]:
```suggestion
   * For **Ubuntu 22.04 and above** you need Packer version 1.8.1 or newer.
```

File: aws/aws-how-to/instances/build-pro-ami-using-packer.rst [hunk 2]
```diff
@@ -26,8 +26,12 @@ If you have an Ubuntu workstation, run:
 
    sudo apt install packer
 
-For other platforms, use the appropriate `install Packer`_ instructions.
+.. note::
 
+   * For building images based on Ubuntu 22.04 and later, you need packer v1.8.1 or newer.
```
Comment (k-dimple) [inline aws/aws-how-to/instances/build-pro-ami-using-packer.rst:31]:
```suggestion
   * For building images based on Ubuntu 22.04 and later, you need Packer version 1.8.1 or newer.
```

File: aws/aws-how-to/instances/build-pro-ami-using-packer.rst [hunk 3]
```diff
@@ -26,8 +26,12 @@ If you have an Ubuntu workstation, run:
...
+   * The version of Packer present in the Ubuntu archives on Ubuntu 22.04 LTS is v1.6.6. 
+   * Packer is no longer available in the Ubuntu repositories on Ubuntu 24.04 LTS.
```
Comment (k-dimple) [inline aws/aws-how-to/instances/build-pro-ami-using-packer.rst:33]:
```suggestion
   * Packer is no longer available in the Ubuntu repositories from Ubuntu 24.04 onwards.
```

**Review by k-dimple — CHANGES_REQUESTED** [review]
Just a couple of nits.

**Review by k-dimple — APPROVED** [review]

**Review by toabctl — APPROVED** [review]

**Pull request merged**